Projects/Contracts/Sources/Contracts.swift

```diff
@@ -114,17 +116,9 @@ extension Contracts {
             } else if case .goToMovingFlow = action {
                 resultJourney(.movingFlow)
             } else if case let .openEditCoInsured(config, fromInfoCard) = action {
```

Comment:
Can we move openEditCoInsured to CoInsuredNavigationAction inside contracts?
(like we have it for the openMissingCoInsuredAlert inside CoInsuredNavigationAction in Contracts?